src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,90 @@
+package duke;
+
+import duke.command.*;
+
+/**
+ * Parser that parses input given by the user.
+ */
+public class Parser {
+
+    private enum Action {
+        Done,
+        Delete,
+        Todo,
+        Deadline,
+        Event,
+        Find,
+    }
```

Comment:
I like how you separated the the possible user actions into enums! This makes it very clear what action the user is taking 👍